<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SeriesScraper</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#renameButton {
	color: rgb(255, 255, 255);
	background-image: url(:/images/button_blue.png);
	border: none;
	outline: none;
}

#renameButton:pressed {
	background-image: url(:/images/button_blue_pressed.png);
}</string>
   </property>
   <widget class="QTableWidget" name="episodeNameTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>980</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>510</y>
      <width>185</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title">
     <string>Serienauswahl</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLineEdit" name="seriesLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>20</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Serie</string>
     </property>
    </widget>
    <widget class="QComboBox" name="seasonComboBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>35</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>20</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Staffel</string>
     </property>
    </widget>
    <widget class="QLabel" name="correctSeriesLabel">
     <property name="geometry">
      <rect>
       <x>116</x>
       <y>40</y>
       <width>20</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>505</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title">
     <string>Ordnerauswahl</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLineEdit" name="pathLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>400</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>425</x>
       <y>35</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Auswahl</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>20</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pfad</string>
     </property>
    </widget>
    <widget class="QLabel" name="correctPathLabel">
     <property name="geometry">
      <rect>
       <x>415</x>
       <y>40</y>
       <width>20</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="renameButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>520</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Umbenennen</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>510</y>
      <width>120</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title">
     <string>Namens-Schema</string>
    </property>
    <widget class="QComboBox" name="nameSchemeComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>104</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
